### ISSUE TITLE
Inaccurate version bounds in  .cabal file

### ISSUE THREAD

**Issue by hvr** [Gabriel439/Haskell-Foldl-Library#54]
Hi again... :-)

...currently with GHC 7.0/base-4.3 & GHC 7.2/base-4.4 (i.e. a `base >= 4.5` lower bound seems appopriate):

```
[1 of 4] Compiling Control.Foldl.Internal ( src/Control/Foldl/Internal.hs, dist/dist-sandbox-771bb167/build/Control/Foldl/Internal.o )
[2 of 4] Compiling Control.Foldl    ( src/Control/Foldl.hs, dist/dist-sandbox-771bb167/build/Control/Foldl.o )

src/Control/Foldl.hs:203:10:
    Could not deduce (Eq (Fold a b))
      arising from the superclasses of an instance declaration
    from the context (Num b)
      bound by the instance declaration at src/Control/Foldl.hs:203:10-32
    Possible fix:
      add (Eq (Fold a b)) to the context of the instance declaration
      or add an instance declaration for (Eq (Fold a b))
    In the instance declaration for `Num (Fold a b)'

src/Control/Foldl.hs:203:10:
    Could not deduce (Show (Fold a b))
      arising from the superclasses of an instance declaration
    from the context (Num b)
      bound by the instance declaration at src/Control/Foldl.hs:203:10-32
    Possible fix:
      add (Show (Fold a b)) to the context of the instance declaration
      or add an instance declaration for (Show (Fold a b))
    In the instance declaration for `Num (Fold a b)'

src/Control/Foldl.hs:338:10:
    Could not deduce (Eq (FoldM m a b))
      arising from the superclasses of an instance declaration
    from the context (Monad m, Num b)
      bound by the instance declaration at src/Control/Foldl.hs:338:10-46
    Possible fix:
      add (Eq (FoldM m a b)) to the context of the instance declaration
      or add an instance declaration for (Eq (FoldM m a b))
    In the instance declaration for `Num (FoldM m a b)'

src/Control/Foldl.hs:338:10:
    Could not deduce (Show (FoldM m a b))
      arising from the superclasses of an instance declaration
    from the context (Monad m, Num b)
      bound by the instance declaration at src/Control/Foldl.hs:338:10-46
    Possible fix:
      add (Show (FoldM m a b)) to the context of the instance declaration
      or add an instance declaration for (Show (FoldM m a b))
    In the instance declaration for `Num (FoldM m a b)'
xcabal: Error: some packages failed to install:
foldl-1.1.4 failed during the building phase. The exception was:
ExitFailure 1
```

**Comment by Gabriel439** [comment]
Fixed by 65e3afb57bfb1c34ea9783ce66d1c00ac7892aae and uploaded to Hackage as `foldl-1.1.5`

**Comment by Gabriel439** [comment]
Also, thanks for all your work on fixing build issues on Hackage!  I really appreciate it :)
